openapi/sms/v1/partner-callbacks.yaml

```diff
@@ -36,7 +36,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/InvalidDeliveryReportRequestErrors'
+                $ref: './common.yaml#/components/schemas/InvalidFormatErrorBatch'
```

Comment:
I am not 100% sure we should make the names of identically structured error objects identical. Maybe giving them a different name for different endpoints would be a good idea. It could be helpful e.g. in case of code generation,